_tutorials/convert-html-site-to-jekyll.md

```diff
@@ -0,0 +1,362 @@
+
+Although websites can have sophisticated features and controls, we'll keep things simple in this tutorial. This tutorial contains the following sections:
+
+* TOC
+{:toc}
+
+## 1. Understand a basic Jekyll site
+
+First let's start with a grounding in the basics. Stripping a Jekyll site down to an extremely basic level will help clarify what happens in a Jekyll site. If you haven't already installed the jekyll gem, [install it]({% link _docs/installation.md %}).
+
+A simple Jekyll site can consist of just 3 files:
+
+```
+├── _config.yaml
+├── _layouts
+│   └── default.html
+└── index.md
+```
+
+Create these 3 files in a folder called `myjekyllsite`. (Put `default.html` inside a folder called `_layouts`.) Then populate the content of the files as follows:
+
+**_config.yml**
+
+```yaml
+markdown: kramdown
+```
+
+**_layouts/default.html**
+
+```html
+<html>
+  <body>
+     {% raw %}{{ content }}{% endraw %}
+  </body>
+</html>
+```
+
+**index.md**
+
+```yaml
+---
+title: My page
+layout: default.html
+---
+
+Some **bold** content.
+```
+
+Now `cd` to `myjekyllsite` and build the site:
+
+```
+jekyll serve
+```
+
+This is a Jekyll site at the most basic level. Here's what is happening:
+
+* The `_config.yml` file contains settings that Jekyll uses as it processes your site. This basic config file tells Jekyll to convert Markdown to HTML using the kramdown filter.
+* Jekyll looks for files with front matter tags (like those in `index.md`) and [processes the files]({% link _tutorials/orderofinterpretation.md %}) (populating site variables, rendering any Liquid, and converting Markdown to HTML).
+* Jekyll pushes the content from all pages and posts into the `{% raw %}{{ content }}{% endraw %}` tags in the layout specified (`default`) in the front matter tags.
+* The processed files get written as `.html` files in the `_site` directory.
+
+With this basic foundation of how a Jekyll site works, you can convert almost any HTML theme into a Jekyll site. The following sections will walk you through a step-by-step tutorial on converting an HTML template into a Jekyll site.
+
+## 2. Establish the Default Layout
+
+Find your HTML theme and save it as a default layout. If you're converting or cloning (with permission to do so) an existing site, you can right-click the page and view the source code. Copy and paste the source code into a file called `default.html` inside a folder called `_layouts`. This will be the layout template for your pages and posts.
+
+Note that in looking for templates, you want the HTML output of the template. If the template has PHP tags or other dynamic scripts, these dynamic elements will need to be converted to HTML or stripped out, since Jekyll handles only static content.
```

Comment:
Jekyll has its own templating system (Liquid) that allows to retrieve dynamic content.